src/main/java/scraping/models/Section.java

```diff
@@ -34,6 +38,10 @@ public Section(int registrationNumber, String sectionCode, String instructor,
     this.recitations = recitations;
   }
 
+  public int getWaitlistTotal() { return waitlistTotal; }
```

Comment:
Can we make this an `Integer` instead of an `int` ? That way we can use the null value to signify that there's no waitlist.

Reply:
Sure can